src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -83,19 +83,51 @@ function WorkspaceInvitePage(props) {
     const excludedUsers = useMemo(() => PolicyUtils.getIneligibleInvitees(props.policyMembers, props.personalDetails), [props.policyMembers, props.personalDetails]);
 
     useEffect(() => {
-        const inviteOptions = OptionsListUtils.getMemberInviteOptions(props.personalDetails, props.betas, searchTerm, excludedUsers);
-
-        // Update selectedOptions with the latest personalDetails and policyMembers information
-        const detailsMap = {};
-        _.forEach(inviteOptions.personalDetails, (detail) => (detailsMap[detail.login] = OptionsListUtils.formatMemberForList(detail)));
-        const newSelectedOptions = [];
-        _.forEach(selectedOptions, (option) => {
-            newSelectedOptions.push(_.has(detailsMap, option.login) ? {...detailsMap[option.login], isSelected: true} : option);
+        let emails = searchTerm.replace(/\s,\s/g, ',').split(',');
+        emails = _.filter(
+            _.map(emails, (word) => word.trim()),
+            (email) => email !== '',
+        );
```

Comment:
Simplification suggestion: 
- If we change the regex to search for any number of spaces around the comma (with `\s*`), then we don't need the `trim()`
- We can use `_.compact(<values>)` to remove falsy values (e.g. `''`)

```suggestion
        const emails = _.compact(searchTerm.replace(/\s*,\s*/g, ',').split(','));
```


```js
› _.compact("aaa    ,    bbb,   cccc ,  ".replace(/\s*,\s*/g, ',').split(','));
// ['aaa', 'bbb', 'cccc']
```

Reply:
Only issue with this is that the leading and trailing spaces will not be removed. E.g. `"    aaaa,bbbb,ccc    "`

Reply:
I will change it to `const emails = _.compact(searchTerm.trim().replace(/\s*,\s*/g, ',').split(','));`

Reply:
Ah yeah good point, your change works!